src/api/endpoints/logged_activities.py

```diff
@@ -342,6 +371,61 @@ def put(cls, logged_activity_id):
         else:
             return response_builder(dict(
                     status='failed',
-                    message='This logged activity is either in-review, approved or already rejected'
-                    ),
-                    406)
+                    message='This logged activity is either in-review,'
+                            ' approved or already rejected'), 406)
+
+
+class LoggedActivityInfoAPI(Resource):
+    """Allows success-ops to request more info on a Logged Activity."""
+
+    decorators = [token_required]
+
+    @classmethod
+    @roles_required(["success ops"])
+    def put(cls, logged_activity_id=None):
+        """Put method for requesting more info on a logged activity."""
+        payload = request.get_json(silent=True)
+
+        if not payload:
+            return response_builder(dict(
+                message="Data for editing must be provided",
+                status="fail"
+            ), 400)
+
+        if not logged_activity_id:
+            return response_builder(dict(
+                status="fail",
+                message="LoggedActivity id must be provided."), 400)
+
+        logged_activity = LoggedActivity.query.get(logged_activity_id)
+
+        if not logged_activity:
+            return response_builder(dict(message='Logged activity not found'),
+                                    404)
+
+        comment = payload.get("comment")
+        if comment:
+            send_email.delay(
+                sender=current_app.config["SENDER_CREDS"],
+                subject="More Info on Logged Activity for {}".format(
+                                    logged_activity.user.society.name),
+                message="Success Ops needs more information on this"
+                        " logged activity: {}.\\n Context: {}."
+                        "\\nClick <a href='{}'>here</a>"
+                        " to view the logged activity and edit the"
+                        " description to give more informaton.".format(
+                            logged_activity.name, comment, request.host_url +
+                            '/api/v1/logged-activities/' +
```

Comment:
This is a link to the backend. You want a link to the frontend. I don't even think it exists yet.